slackbot/dispatcher.py

```diff
@@ -177,3 +185,7 @@ def channel(self):
     @property
     def body(self):
         return self._body
+
+    def docs_reply(self):
+        reply = ['    • `{0}` {1}'.format(v.__name__, v.__doc__ or "") for p, v in iteritems(self._plugins.commands['respond_to'])]
```

Comment:
What is the typical use case of `message.docs_reply()` in the context of a plugin function? IMHO it's not worth a feature to provide something like this.

Also this code would not work because the `PluginManager` is not initialized in the `Message` class.

My suggestion: remove this `Message.docs_reply()` feature.